src/components/mainMenu/MainMenu.jsx

```diff
@@ -77,6 +78,13 @@ const MainMenu = () => {
 
   const setWsId = wsId => setCurrentWsId(wsId);
   const setCountRequests = count => setCount(count);
+  // --begin Mock refresh token
+  const data = {
+    login: 'o99@alt.com',
+    password: 'ssssss'
+  };
+  refreshToken(data);
```

Comment:
Why menu is invoker for this functionality?
It should be started in auth service instance.
Mocked data can be placed there as well.